roles/transformers/tasks/main.yml

```
@@ -108,36 +107,10 @@
         url_username: "{{ nexus_user }}"
         url_password: "{{ nexus_password }}"
 
-    - name: Add ats common vars to setenv file
-      ansible.builtin.blockinfile:
-        path: "{{ config_folder }}/setenv.sh"
-        marker: "# {mark} ATS COMMON ENV VARS"
-        owner: "{{ username }}"
-        group: "{{ group_name }}"
-        block: |
-          export ATS_HOME={{ binaries_folder }}/transform-service
-          export ATS_TENGINE_AIO_HOST={{ ats_tengine_aio_host }}
-          export ATS_SHARED_FS_HOST={{ sfs_host }}
-        insertafter: EOF
-    - name: Add ats dependencies vars to setenv file
-      vars:
-        im_lib: >-
-          {% if pkg_format == 'deb' %}${IMAGEMAGICK_HOME}/lib/x86_64-linux-gnu/{% else %}/usr/lib64/{% endif %}
-        im_cfg: >-
-          {% if pkg_format == 'deb' %}${IMAGEMAGICK_DYN}ImageMagick-{{ imagemagick_version.split('-')[0] -}}{% else %}${IMAGEMAGICK_HOME}{% endif %}
-      ansible.builtin.blockinfile:
-        path: "{{ config_folder }}/setenv.sh"
-        marker: "# {mark} ATS DEPENDENCIES ENV VARS"
-        owner: "{{ username }}"
-        group: "{{ group_name }}"
-        block: |
-          export LIBREOFFICE_HOME={{ libreoffice_home }}
-          export IMAGEMAGICK_HOME={{ imagemagick_home }}
-          export IMAGEMAGICK_DYN={{ im_lib }}
-          export IMAGEMAGICK_CONFIG={{ im_cfg }}/config-Q16HDRI
-          export IMAGEMAGICK_CODERS={{ im_cfg }}/modules-Q16HDRI/coders
-          export IMAGEMAGICK_EXE="{% if pkg_format == 'deb' %}${IMAGEMAGICK_HOME}/bin/{% endif %}convert"
-        insertafter: EOF
+    - name: Delete startup script
```

```suggestion
    - name: Delete deprecated startup script
```

minor: move this task where the `Add AIO startup script` was